packages/@lwc/engine-server/src/polyfills.ts

```diff
@@ -5,27 +5,28 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
 
-import { defineProperties, globalThis } from '@lwc/shared';
+import { defineProperty, globalThis } from '@lwc/shared';
 
 /**
  * The following constructor might be used in either the constructor or the connectedCallback. In
  * order to ensure that the component evaluates, we attach those mock constructors to the global
  * object.
  */
-if (typeof Event !== 'function' && typeof CustomEvent !== 'function') {
+if (typeof Event !== 'function') {
```

Comment:
I would recommend adding a comment explaining that `Event` is defined in Node.js 16+

Reply:
Done!